src/main/java/com/xebialabs/overthere/spi/BaseOverthereFile.java

```diff
@@ -83,12 +83,23 @@ public OverthereFile getFile(String child) {
     @Override
     public void deleteRecursively() throws RuntimeIOException {
         if (isDirectory()) {
+            boolean allSucces = true;
             for (OverthereFile each : listFiles()) {
-                each.deleteRecursively();
+                try {
+                    each.deleteRecursively();
+                } catch (RuntimeIOException rio) {
+                    logger.warn("Unable to delete child "+ each + ". Continue...", rio);
```

Comment:
Use `{}` in the format string instead of concatenating strings.

Reply:
I removed the immediate logging of the stacktrace in favour of adding them as suppressed to the RuntimeIOException that gets thrown, and only log a one-line warning